app/features-json/project_json_controller.rb

```diff
@@ -25,5 +32,80 @@ class ProjectJSONController < AuthenticatedControllerBase
 
       return project_view_model.to_json
     end
+
+    post HOME do
+      REQUIRED_PARAMS = Set["lane", "repo_org", "repo_name", "project_name", "trigger_type"]
+      has_required_params = REQUIRED_PARAMS.subset?(Set.new(@request_payload.keys))
+
+      if has_required_params
```

Comment:
Minor thing, if you use
```ruby
raise "something" unless has_required_params`
```
the whole rest of the method can be moved more to the left 🚔

Reply:
done